hermit-cli/Cargo.toml

```diff
@@ -1,3 +1,21 @@
 [workspace]
 members = [ "crates/*" ]
-resolver = "2"
\ No newline at end of file
+resolver = "2"
+
+[profile.dev]
```

Comment:
Instead of making the dev/debug build tiny, it would be better to adapt `hermit-cli/CMakeLists.txt` to by default do a release build, but with `-DCMAKE_BUILD_TYPE=Debug` do a dev/debug build.

Reply:
Done, but you might need to double check my approach.
- Modified `hermit-cli/CMakeLists.txt` and `/CMakeLists.txt` to introduce a check on CMAKE_BUILD_TYPE. (also introduced CMAKE_BUILD_TYPE_LOWER as it's needed for some paths and for point n°2 below)
- Created two `Hermitfile` variants (debug and release), this might not be the best approach. I couldn't find any option to pass a `VAR/ENV VAR` to be evaluated during the parse of the 'FROM' instruction.

Reply:
I spoke too fast, we can move/copy `hermitfile-parser.wasm` from ` target/wasm32-wasi/debug/hermitfile-parser.wasm`  or `target/wasm32-wasi/debug/hermitfile-parser.wasm`  to `.`  and have one `Hermitfile`.

Do you prefer this ?

Reply:
For now, I prefer the two Hermitfile's over the complexity of a copy. This looks good, thank you!